python/cuxfilter/charts/core/non_aggregate/core_non_aggregate.py

```diff
@@ -29,7 +29,10 @@ class BaseNonAggregate(BaseChart):
     def name(self):
         # overwrite BaseChart name function to allow unique chart on value x
         chart_type = self.chart_type if self.chart_type else "chart"
-        return f"{self.x}_{self.y}_{chart_type}"
+        return (
+            f"{self.x}_{self.y}_{self.aggregate_col or ''}_"
+            f"{self.aggregate_fn or ''}_{chart_type}_{self.title}"
```

Comment:
Just noting that 
```
{self.aggregate_col or ''}_{self.aggregate_fn or ''}_
```
seems like it could end up leaving double underscore `__` in the name (maybe that's fine, just noting in case)

Reply:
yep, was aware of it. The aim is just to have unique names. 

On second thought, might as well be worth it to handle the extra `_` to avoid any future confusions